### PR TITLE
Concat method issue #6

### PR DESCRIPTION
**Pull request by i2r**

### CODE REVIEW

**Comment by i2r** [comment]
@vitkarpov

File: lib/index.js [hunk 1]
```diff
@@ -74,12 +75,30 @@ class BEMClassName {
...
 
     /**
+     * Adds optional string to the chain.
+     * Will be skiped if the argument equals null.
+     *
+     * @param {?String} value
+     * @return {BEMClassName}
+     */
+    concat(value) {
+        if (typeof value !== 'string' && !value) {
```
Comment (vitkarpov) [inline lib/index.js:85]: Seems it should be `||`? Skip the step if the value is not a string **or** empty string?

Comment (vitkarpov) [inline lib/index.js:85]: Nope. It's okay. Not string and not falsy value.

**Comment by vitkarpov** [comment]
👍

**Pull request merged**